src/toil/provisioners/clusterScaler.py

```diff
@@ -265,9 +265,7 @@ def addCompletedJob(self, job, wallTime):
         """
         Adds the shape of a completed job to the queue, allowing the scalar to use the last N
         completed jobs in factoring how many nodes are required in the cluster.
-
-        :param toil.job.JobNode job: The memory, core and disk requirements of the completed job
-
+        :param IssuedJob job: The memory, core and disk requirements of the completed job
```

Comment:
Merge error? The old type hint was correct but the comment was wrong. 'job' is now just a jobNode object not a tuple.